scss/donate.scss

```diff
@@ -1,36 +1,173 @@
 /*@use 'vars';*/
 //temporary until resolve _vars.scss issue
-$cyan: #82ecec;
-$pink: #f27aaa;
-$lavender: #b5a1c4;
-$light-lavender: #cca8c1;
-$ex-light-lavender: #e9e3ed;
-$hero-blue: #6fcbdc;
 
-.donate-button {
-  text-align: center;
-  color: #fff;
-  border: none;
-  background-color: #ff5789;
-  align-items: center;
-  padding: 7px 35px;
-  font-size: 35px;
-  font-weight: bold;
-  border-radius: 6px;
+HERO SECTION BEGINS HERE
+.hero-clouds {
+  position: absolute;
+  width: 100%;
+  height: 100vh;
+  z-index: 1;
+  @media only screen and (max-width: 767px) {
+    width: 834px;
+    height: 663px;
+    left: -100px;
+    top: -26px;
+  }
+
+  @media only screen and (max-width: 1024px) {
+    width: 834px;
+    height: 663px;
+    left: 0px;
+    top: 69px;
+  }
 }
 
-.donate-button:hover {
-  opacity: 75%;
-  text-decoration: none;
-  color: #fff;
-  transition: 0.3s ease;
+.hero {
+  position: relative;
+  overflow: hidden;
+  padding-bottom: 10%;
 }
 
-.b1 {
-  padding-bottom: 40px;
-  padding-top: 20px;
+@keyframes wait {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 0;
+  }
+}
+
+@keyframes fade-in {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+    pointer-events: auto;
+  }
+}
+
+@mixin fade-in($waitTime) {
+  animation: wait $waitTime, fade-in 800ms $waitTime;
+  animation-duration: 2s;
+}
+
+.hero-information {
+  position: relative;
+  z-index: 3;
+  @include fade-in(600ms);
+  .hero-logo {
+    margin-bottom: 15px;
+  }
+
+  @media only screen and (max-width: 767px) {
+    margin-top: -225px;
+  }
+  #technica-logo {
+    padding-top: 5%;
+    margin-top: 10%;
+    width: 600px;
+    @media only screen and (max-width: 767px) {
+      padding-top: 10%;
+    }
+  }
+  #hero-tagline {
+    font-size: 150%;
+    width: 55%;
+    @media only screen and (max-width: 767px) {
+      font-size: 100%;
+      width: 95%;
+    }
+    @media only screen and (min-width: 767px) and (max-width: 1042px) {
+      width: 85%;
+    }
+  }
+}
+
+.hero-action {
+  position: relative;
+  z-index: 2;
+  opacity: 0;
+  animation: fade-in 2s;
+  animation-delay: 1.5s;
+  animation-fill-mode: forwards;
+
+  @media only screen and (min-width: 996px) {
+    padding-top: 115px;
+  }
+
+  @media only screen and (min-width: 1400px) {
+    padding-top: 350px;
+  }
+
+  .register-btn {
```

Comment:
(also rename) donate button big too big & text doesn't scale with size of button on mobile
<img width="500" alt="Screen Shot 2021-06-24 at 12 13 54 AM" src="https://user-images.githubusercontent.com/62481939/123201646-1732b180-d481-11eb-87f2-5b72e326b334.png">